aspnetcore/security/cors/3.1sample/Cors/WebAPI/Controllers/WidgetController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+
+namespace WebAPI.Controllers
+{
+    #region snippet
+    [Route("api/[controller]")]
+    [ApiController]
+    public class WidgetController : ControllerBase
+    {
+        // GET api/values
+        [EnableCors("AnotherPolicy")]
+        [HttpGet]
+        public ActionResult<IEnumerable<string>> Get()
+        {
+            return new string[] { "green widget", "red widget" };
+        }
+
+        #region snippet2
+        // GET api/values/5
+        [EnableCors]        // Default policy.
+        [HttpGet("{id}")]
+        public ActionResult<string> Get(int id)
+        {
+            switch (id)
+            {
+                case 1:
+                    return "green widget";
+                case 2:
+                    return "red widget";
+                default:
+                    return NotFound();
+            }
```

Comment:
Great place to take advantage of C# 8:

```suggestion
            return id switch
            {
                1 => "green widget",
                2 => "red widget",
                _ => NotFound(),
            };
```